apps/etterlatte-behandling/src/main/kotlin/sak/SakService.kt

```diff
@@ -110,15 +95,17 @@ class SakServiceImpl(
     override fun finnEllerOpprettSak(
         fnr: String,
         type: SakType,
-        enhet: String?,
+        overstyrendeEnhet: String?,
         gradering: AdressebeskyttelseGradering?,
+        sjekkEnhetMotNorg: Boolean,
     ): Sak {
-        val enhetFraNorg = finnEnhetForPersonOgTema(fnr, type.tema, type).enhetNr
-        if (enhet != null && enhet != enhetFraNorg) {
-            logger.info("Finner/oppretter sak med enhet $enhet, selv om geografisk tilknytning tilsier $enhetFraNorg")
-        }
-
-        val sak = finnSakerForPersonOgType(fnr, type) ?: dao.opprettSak(fnr, type, enhet ?: enhetFraNorg)
+        val enhet =
+            if (sjekkEnhetMotNorg) {
+                sjekkEnhet(fnr, type, overstyrendeEnhet)
+            } else {
```

Comment:
Denne if-else-blokka er den funksjonelle endringa her, resten er refaktorisering

Reply:
Forresten. Var problemet at kallet mot norg kastet feil, isåfall må vi kanskje wrappe det i try-catch også?

Reply:
Frå migreringservice sender vi sjekkEnhetMotNorg=false, så da gjer vi ikkje oppslaget der i det heile tatt